lib/raven/integrations/rails/middleware/public_exceptions_catcher.rb

```diff
@@ -0,0 +1,22 @@
+module Raven
+  class Rails
+    module Middleware
+      module PublicExceptionsCatcher
+        def self.included(base)
+          base.send(:alias_method_chain, :call, :raven)
+        end
+
+        def call_with_raven(env)
+          request = ActionDispatch::Request.new(env)
+          status  = request.path_info[1..-1].to_i
+
+          if status >= 500
```

Comment:
I don't think we should be adding a non-configurable filter on what exceptions get sent. It sounds to me like the default exception ignoring behavior works fine here.